src/shims/env.rs

```diff
@@ -133,11 +133,15 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
             Ok(cwd) => {
                 // It is not clear what happens with non-utf8 paths here
                 let mut bytes = cwd.display().to_string().into_bytes();
-                // If the buffer is smaller or equal than the path, we return null.
+                // If `size` is smaller or equal than the `bytes.len()`, writing `bytes` using the
+                // `buf` pointer would cause an overflow, the desired behavior in this case is to
+                // return null.
```

Comment:
This should already mention the NULL terminator; without that context it's not clear why equal sizes are a problem.

Reply:
> Could you also rename the "file_manipulation.rs" test to "fs.rs"? That corresponds with the libstd module it tests.

Done in https://github.com/rust-lang/miri/pull/985